aws/resource_aws_launch_template.go

```diff
@@ -1025,6 +1042,17 @@ func getPlacement(p *ec2.LaunchTemplatePlacement) []interface{} {
 	return s
 }
 
+func getHibernationOptions(m *ec2.LaunchTemplateHibernationOptions) []interface{} {
```

Comment:
Rename to `flattenLaunchTemplateHibernationOptions`?

Reply:
while it's different from other resources, i wanted to stick to the naming of other "flatten" functions for Launch Config.

Reply:
done